README.md

```diff
@@ -47,30 +47,39 @@ By default, legitify will check the policies against all your resources (organiz
 You can control which resources will be analyzed with command-line flags namespace and org:
 - `--namespace (-n)`: will analyze policies that relate to the specified resources 
 - `--org`: will limit the analysis to the specified organizations
+- `--scm`: will analyze a specific source code management platform. Possible values are: `github` or `gitlab`
 
 ```
 LEGITIFY_TOKEN=<your_token> legitify analyze --org org1,org2 --namespace organization,member
 ```
 The above command will test organization and member policies against org1 and org2.
 
-## GitHub Requirements
+## Requirements
+### GitHub (Cloud and Enterprise Server)
 1. To get the most out of legitify, you need to be an owner of at least one GitHub organization. Otherwise, you can still use the tool if you're an admin of at least one repository inside an organization, in which case you'll be able to see only repository-related policies results.
-2. legitify requires a GitHub personal access token (PAT) to analyze your resources successfully, which can be either provided as an argument (`-t`) or as an environment variable (`$GITHUB_ENV`).
+2. legitify requires a GitHub personal access token (PAT) to analyze your resources successfully, which can be either provided as an argument (`-t`) or as an environment variable (`LEGITIFY_TOKEN`).
    The PAT needs the following scopes for full analysis:
   ```
   admin:org, read:enterprise, admin:org_hook, read:org, repo, read:repo_hook
   ```
 See [Creating a Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) for more information.  
 Fine-grained personal access tokens are currently not supported because they do not support GitHub's GraphQL (https://github.blog/2022-10-18-introducing-fine-grained-personal-access-tokens-for-github/)
 
-## GitHub Enterprise Support
-You can run legitify against a GitHub Enterprise instance if you set the endpoint URL in the environment variable ``SERVER_URL``:
+### GitHub Enterprise Server
+You can run legitify against a GitHub Enterprise Server instance if you set the endpoint URL in the environment variable ``SERVER_URL``:
 
 ```sh
 export SERVER_URL="https://github.example.com/"
 LEGITIFY_TOKEN=<your_token> legitify analyze --org org1,org2 --namespace organization,member
 ```
 ## GitLab Cloud/Server Support
+1. As mentioned in the previous section, you need to be an owner of at least one GitLab group. Otherwise, you can still use the tool if you're an admin of at least one project inside a group, in which case you'll be able to see only repository-related policies results.
```

Comment:
That's true for GL cloud. But for GL server, don't you need to be an admin on the instance?

Reply:
No, it's okay to be only an owner to provide the same level of collection

Reply:
OK, we'll have to modify it when we'll add server-level policies